common/lib/capa/capa/capa_problem.py

```diff
@@ -143,6 +143,7 @@ def __init__(self, problem_text, id, capa_system, capa_module,  # pylint: disabl
             state (dict): containing the following keys:
                 - `seed` (int) random number generator seed
                 - `student_answers` (dict) maps input id to the stored answer for that input
+                - 'saved_answers' (Boolean) True is answer has been saved since last submit.
```

Comment:
I would expect "saved_answers" to be a dict (like "student_answers"). How about has_saved_answers?

Note also the typo "True is answer".